security/README.md

```diff
@@ -0,0 +1,42 @@
+Security
+========
+
+A guide for practicing safe web.
+
+Think
+-----
+
+Security is important, and you can't practice these guidelines without
+understanding them. Question each guideline before following it yourself and
+start a discussion whenever you're in doubt.
+
+Passwords
+---------
+
+* Use a unique password for every account you create.
+* Don't store passwords in plain text.
```

Comment:
The first bullet point seems like a guide for a user, while the second is for a developer who is building software. I'd find this easier to follow if this guide was split by audience, if so.